src/atoms/Flex/Flex.web.js

```diff
@@ -1,3 +1,76 @@
-const Flex = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Flex = styled(
+  ({
+    alignContent,
+    alignItems,
+    alignSelf,
+    display,
+    flex,
+    flexBasis,
+    flexDirection,
+    flexGrow,
+    flexShrink,
+    flexWrap,
+    justifyContent,
+    order,
+    children,
+    style,
+    ...props
+  }) =>
+    React.cloneElement(children, {
+      ...props,
+      style: { ...style, ...children.props.style },
+    }),
```

Comment:
Once you do the above change you can probably just do this
```suggestion
    React.cloneElement(children, props),
```

Reply:
done